<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerMyCppLoadableModuleWidget</class>
 <widget class="QWidget" name="qSlicerMyCppLoadableModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>110</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="volumeLabel">
       <property name="text">
        <string>Select Volume:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qMRMLNodeComboBox" name="VolumeSelector" native="true">
       <property name="nodeTypes" stdset="0">
        <stringlist>
         <string>vtkMRMLScalarVolumeNode</string>
        </stringlist>
       </property>
       <property name="noneEnabled" stdset="0">
        <bool>true</bool>
       </property>
       <property name="showHidden" stdset="0">
        <bool>false</bool>
       </property>
       <property name="showChildNodeTypes" stdset="0">
        <bool>false</bool>
       </property>
       <property name="baseName" stdset="0">
        <string>VolumeSelector</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="testButton">
     <property name="text">
      <string>Run Logic Test</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="drawButton">
     <property name="text">
      <string>Draw</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qmrmlnodecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
